src/nls/messages_zh-Hans.json

```diff
@@ -259,42 +259,42 @@
-    "dashboard.triggerDetails.webhookInterceptorService": "",
-    "dashboard.triggerTemplate.noParams": "",
-    "dashboard.triggerTemplate.resourceTemplates": "",
-    "dashboard.universalFields.name": "",
-    "dashboard.universalFields.nameInvalid": "",
-    "dashboard.universalFields.namespaceInvalid": "",
-    "dashboard.universalFields.passwordRadioButton": "",
-    "dashboard.universalFields.secretType": "",
-    "dashboard.universalFields.serverURL": "",
+    "dashboard.taskRun.unavailable": "TaskRun 不可用",
+    "dashboard.taskRunParams.name": "名称",
+    "dashboard.taskRunParams.value": "值",
+    "dashboard.taskRuns.errorLoading": "TaskRun 加载失败",
+    "dashboard.tasks.errorLoading": "Task 加载失败",
+    "dashboard.tasksDropdown.empty.allNamespaces": "未找到 Task",
+    "dashboard.tasksDropdown.empty.selectedNamespace": "在 Namespace ''{namespace}'' 中未找到 Task",
+    "dashboard.tasksDropdown.label": "选择 Task",
+    "dashboard.tooltipDropdown.empty": "未找到项目",
+    "dashboard.trigger.noHeaders": "未找到该 Interceptor 的 header。",
+    "dashboard.trigger.noOverlays": "未找到该 Interceptor 的 overlay。",
+    "dashboard.triggerBinding.noParams": "未找到该 TriggerBinding 的参数。",
+    "dashboard.triggerDetails.celInterceptorFilter": "过滤器：",
+    "dashboard.triggerDetails.celInterceptorOverlays": "Overlay：",
+    "dashboard.triggerDetails.interceptorHeader": "Header：",
+    "dashboard.triggerDetails.interceptorName": "名称：",
+    "dashboard.triggerDetails.interceptorNamespace": "Namespace：",
+    "dashboard.triggerDetails.interceptorTitle": "{interceptorNumber}. ({interceptorType}) {interceptorName}",
+    "dashboard.triggerDetails.interceptors": "Interceptor：",
+    "dashboard.triggerDetails.triggerBindings": "TriggerBinding：",
+    "dashboard.triggerDetails.triggerTemplate": "TriggerTemplate：",
+    "dashboard.triggerDetails.webhookInterceptorSecret": "Secret：",
+    "dashboard.triggerDetails.webhookInterceptorSecretKey": "Key：",
+    "dashboard.triggerDetails.webhookInterceptorService": "Service：",
+    "dashboard.triggerTemplate.noParams": "没有找到该模板的参数。",
+    "dashboard.triggerTemplate.resourceTemplates": "资源模版",
+    "dashboard.universalFields.name": "名称",
+    "dashboard.universalFields.nameInvalid": "不得以 - 开头或结尾，且少于 253 个字符，仅包含小写字母、数字或 -。",
+    "dashboard.universalFields.namespaceInvalid": "Namespace 是必须的",
+    "dashboard.universalFields.passwordRadioButton": "密码",
+    "dashboard.universalFields.secretType": "类型",
+    "dashboard.universalFields.serverURL": "服务器 URL",
     "react-intl-formatted-duration.daysUnit": "{value, plural, one {day} other {days}}",
     "react-intl-formatted-duration.duration": "{value} {unit}",
     "react-intl-formatted-duration.hoursUnit": "{value, plural, one {hour} other {hours}}",
     "react-intl-formatted-duration.minutesUnit": "{value, plural, one {minute} other {minutes}}",
     "react-intl-formatted-duration.secondsUnit": "{value, plural, one {second} other {seconds}}"
```

Comment:
The strings displaying duration are not translated. If you could translate these it would be great as we would then have a complete translation for the existing UI in Simplified Chinese.

These strings are displayed in a number of places and are used by a package called [`react-intl-formatted-duration`](https://github.com/en-japan-air/react-intl-formatted-duration#localization).

PipelineRuns / TaskRuns page:
  
![image](https://user-images.githubusercontent.com/2829095/106192217-c950a080-61a3-11eb-9221-eea4a629f722.png)

PipelineRun / TaskRun page on the TaskRun / step details:

![image](https://user-images.githubusercontent.com/2829095/106192636-5d226c80-61a4-11eb-978b-1c7a4a40f80e.png)

Reply:
Fixed. Sorry I have missed it. 😂